app/models/like.rb

```diff
@@ -1,12 +1,13 @@
 class Like < ApplicationRecord
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] Please according the `ERD`, `likes` belongs to `user` and `post` models, kindly remove the associations above and update the association with the post model like so `belongs_to :post`.